src/Uno.UI.Composition/Win2D/Microsoft/Graphics/Canvas/CanvasAlphaMode.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Microsoft.Graphics.Canvas
+{
+	public enum CanvasAlphaMode
```

Comment:
Can we keep these Win2D classes `internal` for now or are they used by end users? Reason is that when we eventually separate Win2D in its own assembly, this will be a binary breaking change